workflow/scripts/rename_contigs.py

```
@@ -0,0 +1,15 @@
+
+sys.stderr = open(snakemake.log[0], "w")
+
+rename_expressions = snakemake.params.expressions
+
+vcf_file = pysam.VariantFile(snakemake.input[0])
+contigs = list(vcf_file.header.contigs)
+vcf_file.close()
+
+with open(snakemake.output[0], "w") as out:
+    for contig in contigs:
+        for expression in snakemake.params.expressions:
```

Ruff (0.8.0)

Undefined name `snakemake`

(F821)